config/locales/es.yml

```diff
@@ -97,7 +98,7 @@ es:
     shared:
       unauthorized:
         explained: No está autorizado para acceder a esta página
-        unauthorized: No autorizado
+        unauthorized: No autorizadoz
```

Comment:
is the last `z` intended?

Reply:
Not really, seems to be missed, I'll remove it